backend/README.md

```diff
@@ -63,3 +55,14 @@ If you want to run app locally, you can use:
 ```sh
 make run # poetry run uvicorn src.main:app --reload
 ```
+
+## Running the tests
+To run tests, you can use:
+```sh
+make test # poetry run pytest
```

Comment:
I would also add info about checking the code coverage on backend:
`pytest --cov`

Reply:
my bad, I'm blind